subsys/bluetooth/services/fast_pair/fp_gatt_service.c

```diff
@@ -757,3 +763,18 @@ static ssize_t write_additional_data(struct bt_conn *conn,
 
 	return res;
 }
+
+int bt_fast_pair_info_cb_register(const struct bt_fast_pair_info_cb *cb)
+{
+	if (!cb) {
+		return -EINVAL;
+	}
+
+	if (fast_pair_info_cb) {
+		return -EALREADY;
+	}
```

Comment:
I am not sure if there would be such a use case but as i guess we won't need to re-register callback for other one in a runtime?

Reply:
Currently, I don't see such a use case. We can always extend the current implementation to support unregister operation with NULL input parameter like it is currently implemented in this Bluetooth API:

```
int bt_conn_auth_cb_register(const struct bt_conn_auth_cb *cb)
{
	if (!cb) {
		bt_auth = NULL;
		return 0;
	}

	if (bt_auth) {
		return -EALREADY;
	}
```

or even support multiple callbacks registrations with the linked list approach and a dedicated unregister API.

Reply:
ok, i guess we can leave that for the future.